sickbeard/providers/scc.py

```diff
@@ -130,12 +130,12 @@ def _get_episode_search_strings(self, ep_obj, add_string=''):
         if self.show.air_by_date:
             for show_name in set(show_name_helpers.allPossibleShowNames(self.show)):
                 ep_string = sanitizeSceneName(show_name) + ' ' + \
-                            str(ep_obj.airdate).replace('-', '|')
+                            str(ep_obj.airdate).replace('-', '.')
                 search_string['Episode'].append(ep_string)
         elif self.show.sports:
             for show_name in set(show_name_helpers.allPossibleShowNames(self.show)):
                 ep_string = sanitizeSceneName(show_name) + ' ' + \
-                            str(ep_obj.airdate).replace('-', '|') + '|' + \
+                            str(ep_obj.airdate).replace('-', '.') + '|' + \
```

Comment:
I have a feeling this will not work either. As SCC will treat it as a literal rather than an 'OR' operation as is intended.